.github/changelog/1802-from-description

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: changed
+
+Screen options in the Activitypub settings page are now filterable.
```

Comment:
Is it about filters?

```suggestion
Screen options in the ActivityPub settings page are now filterable.
```

Reply:
How did that get in there? I didn't specify a changelog in the PR description?!

That's from an old PR